protocol/docker-compose.yml

```diff
@@ -17,10 +17,7 @@ services:
       - "17e5e45691f0d01449c84fd4ae87279578cdd7ec@dydxprotocold0:26656,b69182310be02559483e42c77b7b104352713166@dydxprotocold1:26656,47539956aaa8e624e0f1d926040e54908ad0eb44@dydxprotocold2:26656,5882428984d83b03d0c907c1f0af343534987052@dydxprotocold3:26656"
       - --bridge-daemon-eth-rpc-endpoint
       - "${ETH_RPC_ENDPOINT}"
-      - --dd-error-tracking-format
       - "true"
-      - --max-daemon-unhealthy-seconds
```

Comment:
These flags must be removed for the entrypoint to work on `v2` binaries. Without these flags `--max-daemon-unhealthy-seconds` or `--panic-on-daemon-failure-enabled`, localnet nodes might panic (since v3.0.0) after 5 minutes if the bridge/price daemon is flaky. Upgrading (forcing a restart) within 5 minutes avoids this issue. `dev` and `staging` should be fine.